src/OpenTK/Input/JoystickDevice.cs

```diff
@@ -263,17 +263,6 @@ internal JoystickAxisCollection(int numAxes)
         }
 
         /// <summary>
-        /// Gets a System.Single indicating the absolute position of the JoystickAxis.
```

Comment:
Is removing this indexer necessary? Could it instead be altered to just go by an integer index?